frontend/src/features/Reportings/components/ReportingLayer/Reporting/style.ts

```diff
@@ -62,7 +62,8 @@ const selectedReportingStyleFactory = (color, fillColor) => [
       color,
       lineCap: 'square',
       lineDash: [2, 8],
-      width: 5
+      // TODO (21/08/2024) : Vérifier l'épaisseur du trait avec Adeline
+      width: 4
```

Comment:
pourquoi ces changements?

Reply:
C'est un choix personnel : j'ai noté de vérifier avec Adeline car je trouvais le visuel assez étrange